### DISCUSSION TITLE
I'm not sure if bug or features - Levelling too fast and lots of good drops

### DISCUSSION THREAD

**Discussion by Littlebob86** [diasurgical/DevilutionX#6193] [Q&A] [answered]
So I am in the first floor, and have leveled up like 6 times and got already like 3 book drops for spells and stuff. I'm only playing at the 3rd of 4 difficulties. I couldn't see any settings that I might have adjusted by accident? Any idea what's going on?

**Comment by AJenbo** [comment]
There are only 3 difficulties. Are you sure you are not playing some other mod? Also what version are you running?

If you are playing using a new hero then playing at high difficulties would indeed cause you to level up very fast as it is only intended for high level heros and will give a lot of XP for each kill.

**Reply by Littlebob86** [reply]
Sorry, 2nd one. Nightmare. I'm playing sorcerer. And I'm actually level 9, and only like not even 1/4 through the first floor.

**Reply by AJenbo** [reply]
Ok, that would be normal. Usually you would be at least level 20 before starting on Nightmare.

**Reply by Littlebob86** [reply]
But enemies are still dying to like, 1x level 1 firebolt?

**Reply by AJenbo** [reply, marked as answer]
That's how the game was designed back in 1996

**Reply by Trihedraf** [reply]
How did he get into a nightmare game before level 20 that doesn’t work in DevX

**Reply by AJenbo** [reply]
Because he is playing Single player

**Reply by Littlebob86** [reply]
I downloaded the game from some website (I do own the original, but nfi idea where the disk would be). I didn't buy it from gog. Could it be something weird from that? It didn't say anythign about trainers or cheat engines or anything

**Comment by Littlebob86** [comment]
These 2 images are my xp after killing 1 skeleton.. Like.. 30k xp..
![image](https://github.com/diasurgical/devilutionX/assets/25976048/847d2241-2853-45f6-ba4f-194029da37b4)

![image](https://github.com/diasurgical/devilutionX/assets/25976048/ea274571-7bfb-4c55-89c1-a6486d38b678)

**Reply by AJenbo** [reply]
Much more interested in what version you are running...

**Reply by Littlebob86** [reply]
How can I find out?

**Reply by Littlebob86** [reply]
Dex v 1.4.1

**Reply by AJenbo** [reply]
it's in the lower corner of the main menu, or press `v` ingame, or right click the exe and select properties and see the version there like most other Windows applications, or check the version where you downloaded it.

**Reply by Littlebob86** [reply]
Dex v 1.4.1

**Reply by AJenbo** [reply]
Ok, well the only thing I can really say then is that in 1.5.0 (coming in a couple of days) we will have fixed a few things that made Nightmare easier then it was originally intended.

**Reply by Littlebob86** [reply]
Well thank you so much for all of your replies! I'll wait until 1.5 comes out and start again :)

**Reply by AJenbo** [reply]
Your basically power leveling, it was not originally the intention for player to be able to jump in to Nightmare, and never even intended to be accessible in Single Player (where things are easier). That was only possible because of a bug, and later in the Hellfire expansion which was not made by the original developers. Nightmare and Hell also saw very little testing.

**Reply by AJenbo** [reply]
The difficulty will take a step clime after the first couple of levels.

**Comment by Littlebob86** [comment]
Thank you so much for your time :)


-------- Original message --------
From: Anders Jenbo ***@***.***>
Date: 10/6/23 19:57 (GMT+08:00)
To: diasurgical/devilutionX ***@***.***>
Cc: Littlebob86 ***@***.***>, Author ***@***.***>
Subject: Re: [diasurgical/devilutionX] I'm not sure if bug or features - Levelling too fast and lots of good drops (Discussion #6193)


The difficulty will take a step clime after the first couple of levels.

**Reply by AJenbo** [reply]
No worries, hope you have fun playing the game.
